tests/integration/conftest.py

```diff
@@ -171,7 +171,7 @@ def unregister(self):
 
         return self.shell(command)
 
-    def set_up_requirements(self):
+    def set_up_requirements(self, set_to_stage=False):
```

Comment:
I'd use a different name
```suggestion
    def set_up_requirements(self, use_staging_cdn=False):
```

Reply:
renamed